### ISSUE TITLE
Desktop clients

### ISSUE THREAD

**Issue by trymeouteh** [AgoraDesk-LocalMonero/agoradesk-app-foss#224]
Please consider compiling the app for Windows, MacOS and Linux and have the app run in the background in the system tray. This will be useful for buyers and sellers to have the app running on their computers in the background to be notified of a trade without having to keep a tab open to the website in their browser or even have their browser running in the background.

**Comment by sergdeus** [comment]
Yes, it is in our plans. I believe we can start at the end of May. For Mac, it is already available.

**Comment by trymeouteh** [comment]
Please release it on Windows as a scoop package and as a flatpak and appimage on Linux.
